File: Level1/Materal.Workflow/Materal.Workflow/StepDatas/StartStep/StartStepData.cs
using Newtonsoft.Json;
using System.Text.Json.Serialization;

namespace Materal.Workflow.StepDatas
{
    /// <summary>
    /// 开始节点数据
    /// </summary>
    public class StartStepData : StepData, IStepData
    {
        /// <summary>
        /// 下一步
        /// </summary>
        [System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public IStepData? Next { get; set; }
    }
}

File: Level1/Materal.Workflow/Materal.Workflow.Vue/src/scripts/StepDatas/StartStepData.ts
import { IStepData } from "./Base/IStepData";
import { StepData } from "./Base/StepData";

export class StartStepData extends StepData {
    public Next?: IStepData;
    constructor(id: string) {
        super(`${StartStepData.name}`, "开始节点", id);
    }
}